parsers/fasta/fasta.go

```diff
@@ -0,0 +1,165 @@
+package fasta
+
+import (
+	"bufio"
+	"bytes"
+	"compress/gzip"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+/******************************************************************************
+Apr 25, 2021
+
+FASTA Parser begins here
+
+Many thanks to Jordan Campbell (https://github.com/0x106) for building the first
```

Comment:
I heavily refactored Jordan's stuff a while back so I guess I'm also responsible for this.